src/init/register_configure_agent.sh

```diff
@@ -208,22 +193,15 @@ set_vars () {
     export WAZUH_KEY
     export WAZUH_PEM
 
-    WAZUH_MANAGER_IP=$(launchctl getenv WAZUH_MANAGER_IP)
-    WAZUH_NOTIFY_TIME=$(launchctl getenv WAZUH_NOTIFY_TIME)
-    WAZUH_AUTHD_SERVER=$(launchctl getenv WAZUH_AUTHD_SERVER)
-    WAZUH_AUTHD_PORT=$(launchctl getenv WAZUH_AUTHD_PORT)
-    WAZUH_PASSWORD=$(launchctl getenv WAZUH_PASSWORD)
-    WAZUH_GROUP=$(launchctl getenv WAZUH_GROUP)
-    WAZUH_CERTIFICATE=$(launchctl getenv WAZUH_CERTIFICATE)
-    WAZUH_KEY=$(launchctl getenv WAZUH_KEY)
-    WAZUH_PEM=$(launchctl getenv WAZUH_PEM)
+    if [ -r "${WAZUH_MACOS_AGENT_DEPLOYMENT_VARS}" ]; then
+        . ${WAZUH_MACOS_AGENT_DEPLOYMENT_VARS}
+        rm -rf "${WAZUH_MACOS_AGENT_DEPLOYMENT_VARS}"
```

Comment:
This should go in the `unset_vars` function.

Reply:
The motivation for this rm placement was to remove the file as fast as possible. In this case, once the vars are set the file can be removed